pombola/writeinpublic/client.py

```diff
@@ -35,6 +35,28 @@ def __init__(self, params, person):
         self.person = person
 
 
+class Person(RecipientMixin, object):
+    def __init__(self, params, adapter):
+        self.id = params['id']
+        keys = [
+            "created_at",
+            "email",
+            "id",
+            "name",
+            "popit_id",
+            "popit_url",
+            "resource_uri",
+            "updated_at"
+        ]
+        for key in keys:
+            if key == 'created_at' or key == 'updated_at':
+                setattr(self, key, parse_datetime(params.get(key)))
+            setattr(self, key, params.get(key))
+
+        self._params = params
+        self.adapter = adapter
```

Comment:
I'm not totally sure what's happening here - are we generating a duck-typed pombola core person from a a recipient? or am I being dim?

what purpose does this class serve?

Reply:
I just have the suspicion we're mixing in a concept here that doesn't belong in the writeinpublic client and might not need to exist

Reply:
I'm so confused, sorry :) what's an adapter? I just don't understand the relevance of this code to the change and you've probably figured this all out